index.html

```diff
@@ -291,65 +454,198 @@ <h3>
+      <div>
+        The <dfn>Text record</dfn> is a <a>well-known type</a> that is defined
+        in the [[NDEF-TEXT]] specification.
+        The <a>TNF field</a> is `1` and the <a>TYPE field</a> is "`T`" (`0x54`).
+        The first byte of the <a>PAYLOAD field</a> is a status byte, followed
+        by the ISO/IANA language code in US-ASCII encoding.
+        The rest of the payload is the actual text, encoded either in UTF-8 or
+        UTF-16, as indicated by the status byte as follows:
+        <ul>
+          <li>Bits 0 to 5 define the length of the IANA language code.</li>
+          <li>Bit 6 is `0`.</li>
+          <li>
+            If bit 7 if set, means the payload is encoded in UTF-8, otherwise in
+            UTF-16.
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section> <h5>URI record</h5>
+      <p>
+        <dfn>URI record</dfn> is defined in [[NDEF-URI]].
+        The <a>TNF field</a> is `1` and the <a>TYPE field</a> is "`U`" (`0x55`).
+        The first byte of the <a>PAYLOAD field</a> is a URI identifier code,
+        in fact an index in an abbreviation table where the values are prepended
+        to the rest of the URI. For instance the value `0` denotes no prepending,
+        `1` denotes "`http://www.`", `0x04` denotes "`https://`"" and so on.
+        The rest of the payload contains the rest of the URI as a UTF-8 string
+        (and if the first byte is `0`, then it denotes the whole URI).
+      </p>
+      <p>
+        The URI is defined in [[RFC3987]] and in fact is a UTF-8 encoded IRI
+        that can be a URN or a URL.
+      </p>
+    </section>
+
+    <section> <h5>Smart poster record</h5>
     <p>
-      Other known RTD types supported by the platform and not Web NFC are
-      "`ac`" (`0x61`, `0x63`) for alternative carrier, "`Hc`" (`0x48`, `0x63`) for
-      handover carrier, "`Hr`" (`0x48`, `0x72`) for handover request, and "`Hs`"
-      (`0x48`, `0x73`) for handover select.
+      <dfn>Smart poster</dfn> is defined in [[NDEF-SMARTPOSTER]] as an NDEF
+      record that contains an <a>NDEF message</a> as payload, which may contain
+      several records: a mandatory <a>URI record</a> that refers to a content,
+      and additional optional records related to the content: a title record
+      (a <a>Text record</a>), one or more icon records, a type record, a size
+      record and an action record.
     </p>
     <p>
-      An <dfn>NFC handover</dfn> defines RTD and the
-      corresponding message structure that allows negotiation and activation of
-      an alternative communication carrier, such as Bluetooth or WiFi.
-      The negotiated communication carrier would then be used (separately) to
-      perform certain activities between the two devices, such as sending photos
-      to the other device, printing to a Bluetooth printer or streaming video to
-      a television set.
+      Icon records are <a>MIME type record</a>s. Readers SHOULD select one icon
```

Comment:
of MIME type "image/ ?

Reply:
No, one icon record. Will specify like that.